spec/fixy/formatter/signed_amount_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe Fixy::Formatter::SignedAmount do
+  let(:proxy) do
+    Class.new do
+      include Fixy::Formatter::SignedAmount
+    end.new
+  end
+
+  describe '#format_signed_amount' do
+    subject { proxy.format_signed_amount(input, length) }
+
+    let(:length) { 10 }
+
+    [
+      # [nil,     6,  '00000+'],
+      # ['',      6,  '00000+'],
```

Comment:
You could uncomment these if you update
https://github.com/Gusto/fixy/blob/master/lib/fixy/formatter/signed_amount.rb#L14
from
```ruby
sign = input >= 0 ? '+' : '-'
```
to
```ruby
sign = input.to_f >= 0 ? '+' : '-'
```